### PR TITLE
added a limit for laps

### PR DESCRIPTION
**Pull request by sahilsharma356**
added a limit for laps at 99 laps. When the limit is reached, the lap button is removed. Upon reset, the lap button is introduced again.

### CODE REVIEW

File: app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt [hunk 1]
```diff
@@ -29,6 +29,7 @@ class StopwatchFrag : Fragment() {
     private var lapCount = 0
     private var prevTime = 0.toLong()
     private lateinit var laps: ArrayList<LapObject>
+    private var maxLapReached = 0
```
Comment (harigovind1998) [inline app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt]: make this a bool u fool

Comment (harigovind1998) [reply app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt]: you can also do private lateinit var maxLapReached and initialize the var inside the onCreate() function. doesn't make sense to initialize it here and do the same thing in onCreate()

File: app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt [hunk 2]
```diff
@@ -95,6 +98,11 @@ class StopwatchFrag : Fragment() {
 
     private fun lap() {
         lapCount += 1
+
+        if (lapCount >= 99){
```
Comment (harigovind1998) [inline app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt]: Throw an error if its >99 since it should never be < 99

**Comment by harigovind1998** [comment]
Pull in develop changes before merging. Have fun with merge conflicts.

**Review by SVD6 — APPROVED** [review]

**Pull request merged**